### PR TITLE
MNTOR-2424: manually resolved exposures in fixed tab

### PR DESCRIPTION
**Pull request by mansaj**
# References: 
Jira: MNTOR-2424

# Description
Manually resolved exposure were not counted in fixed tab.

# Screenshot (if applicable)

Not applicable.

# How to test
* free scan
* Proceed to manually fix “Info for sale” exposures;
* Observe charts from Action needed and Fixed tabs;
* The Fixed tab should correctly reflect manually resolved items

# Checklist (Definition of Done)
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.
- [x] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

**Comment by mansaj** [comment]
This PR is branched off of https://github.com/mozilla/blurts-server/pull/3701
Will merge after https://github.com/mozilla/blurts-server/pull/3701 is merged

**Review by Vinnl — APPROVED** [review]
Thanks, much better than my attempt! This might also be a good candidate for a unit test to verify that the calculations indeed incorporate manually resolved exposures?

(Also, make sure to align with @codemist before merging, since it'll affect her PR.)

**Pull request merged**
